src/core/task10.js

```diff
@@ -1,4 +1,4 @@
-export var obj = {
+export default var obj = {
```

Comment:
1. You should create an object with 'new' operator 
2. You should process the cases when a user can pass a string or does not pass anything